torchft/manager.py

```diff
@@ -670,7 +695,11 @@ def state_dict(self) -> Dict[str, int]:
         Returns:
             the state dict for this manager
         """
-        return {"step": self._step, "batches_committed": self._batches_committed}
+        return {
+            "step": self._step,
+            "batches_committed": self._batches_committed,
+            "commit_failures": self._commit_failures,
```

Comment:
This is an interesting case that I hadn't considered -- I think it might be useful since it can avoid out of sync issues but I'm also a tad concerned if we write it out to persistent storage and then load from checkpoint. Fresh startup should likely restart this count

Thoughts on leaving this out for now?